### ISSUE TITLE
aws-kinesisfirehose-alpha: Needs support for `ca-west-1`

### ISSUE THREAD

**Issue by parvahuja** [aws/aws-cdk#29874]
### Describe the bug

Creating a DeliveryStream in `ca-west-1` fails at synth time with `Error: Mapping doesn't contain top-level key ca-west-1`

### Expected Behavior

Configured constructs in a stack set to deploy to `ca-west-1` using `aws-cdk-lib` 2.134 and `@aws-cdk/aws-kinesisfirehose-alpha` 2.134-alpha. Expected a clean compile.

### Current Behavior

```
Error: Mapping doesn't contain top-level key 'ca-west-1'
...
```

when attempting to compile.

### Reproduction Steps

Just take any normal code example of `DeliveryStream` like your unit tests and synth it in `ca-west-1`.

### Possible Solution

`aws-kinesisfirehose-alpha` uses a mapping of CIDR blocks (https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts#L768) per region. This list is incomplete (it does not include `ca-west-1`).

### Additional Information/Context

Basically identical issue to https://github.com/aws/aws-cdk/issues/27518, but for another region and the (now) newest CDK

### CDK CLI Version

2.134 

### Node.js Version

v18.18.0

### OS

Amazon Linux 2

### Language

TypeScript

### Language Version

TypeScript (4.9.4)

**Comment by khushail** [comment]
@parvahuja , thanks for reaching out. We have a similar issue filed with CDK Team internally, P125405005. Team is working on this.
